platform/windows/joypad_windows.cpp

```diff
@@ -529,7 +529,7 @@ void JoypadWindows::load_xinput() {
 		return;
 	}
 
-	XInputGetState_t func = (XInputGetState_t)GetProcAddress((HMODULE)xinput_dll, "XInputGetState");
+	XInputGetState_t func = (XInputGetState_t)GetProcAddress((HMODULE)xinput_dll, (LPCTSTR)100);
```

Comment:
Note: This function should be available in XInput 1.3+., but I'm not sure about old XInput 9.1.0. So it should be either tested with this version on Windows 7, or version check added to use `100`/`XInputGetStateEx` only if 1.3/1.4 dll used.

Reply:
Update: Seems like 9.1.0 use 1.0 API, so version check is required.

```diff
	xinput_set_state = &_xinput_set_state;
+	bool legacy_xinput = false;
	xinput_dll = LoadLibrary("XInput1_4.dll");
	if (!xinput_dll) {
		xinput_dll = LoadLibrary("XInput1_3.dll");
		if (!xinput_dll) {
			xinput_dll = LoadLibrary("XInput9_1_0.dll");
+			legacy_xinput = true;
		}
	}

	if (!xinput_dll) {
		print_verbose("Could not find XInput, using DirectInput only");
		return;
	}

-	XInputGetState_t func = (XInputGetState_t)GetProcAddress((HMODULE)xinput_dll, (LPCTSTR)100);
+	XInputGetState_t func = (XInputGetState_t)GetProcAddress((HMODULE)xinput_dll, legacy_xinput ? "XInputGetState" : (LPCTSTR)100);
	XInputSetState_t set_func = (XInputSetState_t)GetProcAddress((HMODULE)xinput_dll, "XInputSetState");
```